src/server/models/list.ts

```diff
@@ -90,10 +95,14 @@ export async function createList(listData: {
         },
       },
       jsonData: {
-        users,
         relatedLinks: getRelatedLinks(listData.serviceType),
         createdBy: listData.createdBy,
       },
+      users: {
+        connect: {
+          email: listData.user,
```

Comment:
since there's a possibility that `listData.user` is not all lowercase, it's safer to do any of these:
- use `user` which has been lowercased
- use the value from `createUserFromEmail`, but would mean `createUserFromEmail` should return the created user